decidim-elections/app/packs/src/decidim/elections/trustee/key_ceremony.js

```diff
@@ -28,13 +25,18 @@ $(() => {
   const getStepRow = (step) => {
     return $(`#${step.replace(".", "-")}`);
   };
+  const TRUSTEE_AUTHORIZATION_EXPIRATION_TIME_IN_HOURS = 2;
 
   // Data
   const bulletinBoardClientParams = {
     apiEndpointUrl: $keyCeremony.data("apiEndpointUrl")
-  }
-  const electionUniqueId = `${$keyCeremony.data("authoritySlug")}.${$keyCeremony.data("electionId")}`
-  const authorityPublicKeyJSON = JSON.stringify($keyCeremony.data("authorityPublicKey"))
+  };
+  const electionUniqueId = `${$keyCeremony.data(
```

Comment:
Just a personal opinion but I think the formatted syntax is harder to read. 😊

Reply:
Yup I agree! Prettier is weird sometimes haha